Fix example that is too specific to AWS provider (#764)
hashicorp/terraform-docs-common: website/docs/cloud-docs/registry/publish-providers.mdx

}
```

Use the [Create a Provider Version endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-version) to create a version for your provider. Replace `TOKEN` in the `Authorization` header with your HCP Terraform API token, and replace both instances of `ORG_NAME` with the name of your organization. Replace `TF_PROVIDER_NAME` in the URL path with your terraform provider name

```shell-session
$ curl \
  --header "Authorization: Bearer $TOKEN" \
  --header "Content-Type: application/vnd.api+json" \
  --request POST \

>   --data @payload.json \

Reviewer: Ooops!
```suggestion
  --data @version.json \
```

}
```

Use the [Create a Provider Version endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-version) to create a version for your provider, and replace the following values:
1. Replace `TOKEN` in the `Authorization` header with your HCP Terraform API token.
1. Replace both instances of `ORG_NAME` with the name of your organization.
1. Replace `TF_PROVIDER_NAME`  with your provider name.

> 

Reviewer: ```suggestion
```

}
```

> Use the [Create a Provider Version endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-version) to create a version for your provider. Replace `TOKEN` in the `Authorization` header with your HCP Terraform API token, and replace both instances of `ORG_NAME` with the name of your organization. Replace `TF_PROVIDER_NAME` in the URL path with your terraform provider name

Reviewer: ```suggestion
Use the [Create a Provider Version endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-version) to create a version for your provider, and replace the following values:
1. Replace `TOKEN` in the `Authorization` header with your HCP Terraform API token.
1. Replace both instances of `ORG_NAME` with the name of your organization.
1. Replace `TF_PROVIDER_NAME`  with your provider name.
```

Reviewer: Updating to a table for easier scanning, but love this contribution!
